frontend/package.json

```diff
@@ -143,7 +143,7 @@
     "decode-uri-component@^0.2.0": "0.2.1",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "@types/react": "18.2.6",
+    "@types/react": "18.2.18",
```

Comment:
Do we need the dependency if the overrides if it's the same version we're using already?